src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java

```diff
@@ -107,6 +107,12 @@
   public static final String TABLE_BLACKLIST_DEFAULT = "";
   private static final String TABLE_BLACKLIST_DISPLAY = "";
 
+  public static final String INCREMENTING_COLUMN_USE_PRIMARY_KEY_CONFIG = "incrementing.column.use.primary.key";
```

Comment:
Here's another idea for how to expose this -- a mode that is `incrementing-primary-key` instead of a separate flag. I'm not sure if that's necessarily better -- I think we may end up with a bunch of combinations since we already have combinations of timestamps & primary keys.

So to make sure I understand the motivation here -- is the reason for the addition that you a) want to use `incrementing` mode, b) have many tables, and c) have primary key columns on those tables named differently? So this option allows you to capture in a smaller set of configs (or smaller set of connectors) all tables at once while the existing configs do not?